### PR TITLE
chore: update readme

### PR DESCRIPTION
**Pull request by 43081j**
just to add a browser example and fix up some bits

cc @stramel

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,29 +12,37 @@ Based on Wagner-Fischer dynamic programming algorithm, optimized for speed and m
...
+$ npm i -S js-levenshtein-esm
 ```
 
 
 ## Usage
 
+In Node:
+
 ```js
 const levenshtein = require('js-levenshtein');
```
Comment (stramel) [inline README.md]: `js-levenshtein-esm`

**Review by stramel — CHANGES_REQUESTED** [review]
Should probably add the npm badge and travis badge

**Comment by 43081j** [comment]
@stramel updated it , added badges and fixed perf part of readme

File: README.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+[![Build Status](https://travis-ci.com/43081j/js-levenshtein.svg?branch=master)](https://travis-ci.com/43081j/js-levenshtein)
+[![npm version](https://img.shields.io/npm/v/js-levenshtein-esm.svg?style=flat)](https://npmjs.org/package/js-levenshtein-esm "View this project on npm")
+
 # js-levenshtein-esm [![Build Status](https://travis-ci.com/43081j/js-levenshtein.svg?branch=master)](https://travis-ci.com/43081j/js-levenshtein)
```
Comment (stramel) [inline README.md]: You have duplicate travis badge

File: README.md [hunk 3]
```diff
@@ -41,32 +44,33 @@ levenshtein('kitten', 'sitting'); // 3
...
-$ npm run bench
+$ npm run test:perf
 
                       50 paragraphs, length max=500 min=240 avr=372.5
-             162 op/s » js-levenshtein
-              98 op/s » talisman
-              94 op/s » levenshtein-edit-distance
-              85 op/s » leven
-              39 op/s » fast-levenshtein
+wait � fast-levenshtein             62 op/s � fast-levenshtein
```
Comment (stramel) [inline README.md]: can you drop the placeholder outputs? 

```
wait » fast-levenshtein             62 op/s » fast-levenshtein
```

should be

```
             62 op/s » fast-levenshtein
```

**Review by stramel — CHANGES_REQUESTED** [review]

**Review by stramel — APPROVED** [review]

**Pull request merged**